kernel_cache_dropper/kernel-cache-drop-websvc.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import subprocess
+import logging
+import cherrypy
+import atexit
+import os
+
+logging.basicConfig(filename='/tmp/dropcache.log', level=logging.DEBUG)
+logger = logging.getLogger('dropcache')
+logger.setLevel(logging.INFO)
+
+def flush_log():
+    logging.shutdown()
+
+atexit.register(flush_log)
+
+class DropKernelCache(object):
+
+    @cherrypy.expose
+    def index(self):
+        return "Hi there\n"
+
+    @cherrypy.expose
+    def DropKernelCache(self):
+        logger.info('asked for cache drop')
+        try:
+            os.sync()
+            logger.info('completed sync call')
+            with open('/proc_sys_vm/drop_caches','a') as dcf:
+                dcf.write('3\n')
+            logger.info('completed cache drop')
+        except FileNotFoundError:
+                logger.error('/proc_sys_vm/drop_caches not found')
```

Comment:
This one is returning 200 OK even if /proc_sys_vm/drop_caches doesn't exist.

Reply:
@rsevilla87  wow, it never occurred to me that it wouldn't exist.   I was only concerned that the pod wouldn't have access to it.   But if the file doesn't exist, then the exception won't be caught and the program will exit.    Are you sure that you passed the pod IP to it?   request_cache_drop.py does absolutely nothing unless it has pod IPs passed to it.

Reply:
It always "return 'SUCCESS'" regardless a exception is caught or not :)

Reply:
I think the suggestion is just to pass a 404 if we hit the `FileNotFoundError`